Examples/Examples.playground/Pages/00ToC.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,20 @@
+/*:
+ ## SwifterSwift Examples
+ 
+ SwifterSwift is a library of **over 500 properties and methods**, designed to extend Swift's functionality and productivity, staying faithful to the original API design guidelines.
+ 
+ You can find examples of some extenstions and try them out in this playground.
+ 
+ Here are list of extensions:
+ 
+ * [SwiftStdlib extensions](01SwiftStdlibExtensions)
+ * [Foundation extensions](02FoundationExtensions)
+ * [UIKit extensions](03UIKitExtensions)
+ * [AppKit extensions](04AppKitExtensions)
+ * [CoreGraphics extensions](05CoreGraphicsExtensions)
+ * [CoreLocation extensions](06CoreLocationExtensions)
+ * [MapKit extensions](07MapKitExtensions)
+ * [SpriteKit extensions](08SpriteKitExtensions)
+ * [Misc. extensions](09MiscExtensions)
+ 
```

Comment:
Maybe we can add a section here with a message of "make sure you build the SwifterSwift project before trying to run the playgrounds".